recipes/pystan/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "pystan" %}
+{% set version = "2.8.0.2" %}
+{% set sha256sum = "5e38e4cc7a71c299491eb7a4d7a82fed1fe44d6464481a65eac0c42c906a0cf7" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+  patches:
+    - patches/0001-Backport-windows-build-flags.patch
+    - patches/0002-Disable-boost-quadmath.patch
+    - patches/0003-Do-not-include-msinttypes-on-vc10-or-later.patch
+
+build:
+  number: 0
+  # Skip 32-bit builds for now.
+  skip: True  # [not x86_64]
```

Comment:
The linter failure is happening b/c of this preprocessing selectors. I am not sure in what version of `conda-build` that was introduced but conda-smithy cannot handle that right now. Since we only build 32-bits on Windows I suggest you change that to `[win32]` instead.